TUTORIAL-3.md

```diff
@@ -14,214 +14,236 @@ Suppose that we want our hello world app from tutorial 2 to insert a row into an
 
 Traveling Ruby provides a specific version of the sqlite3 gem. See [the Traveling Ruby Amazon S3 bucket](https://traveling-ruby.s3-us-west-2.amazonaws.com/list.html). For version 20141215-2.1.5, version 1.3.9 is supplied. So we add the following to our Gemfile:
 
-    gem 'sqlite3', '1.3.9'
+```Bash
+gem 'sqlite3', '1.3.9'
+```
 
 Let's also modify hello.rb to do what we want:
 
-    #!/usr/bin/env ruby
-    require 'faker'
-    require 'sqlite3'
+```Ruby
+#!/usr/bin/env ruby
+require 'faker'
+require 'sqlite3'
 
-    db = SQLite3::Database.new("hello.sqlite3")
-    db.execute("create table if not exists foo (name varchar(255))")
-    db.execute("insert into foo values ('hello world')")
-    db.close
-    puts "Hello #{Faker::Name.name}, database file modified."
+db = SQLite3::Database.new("hello.sqlite3")
+db.execute("create table if not exists foo (name varchar(255))")
+db.execute("insert into foo values ('hello world')")
+db.close
+puts "Hello #{Faker::Name.name}, database file modified."
+```
 
 Then install your gem bundle:
 
-    $ bundle install
+```Bash
```

Comment:
I don't think highlighting this is a good idea because the highlighter breaks at the prompt sign ($).